provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/client/PpmsClient.java

```diff
@@ -12,6 +13,9 @@
   /** Return the parameters of the failed search. */
   ProviderContacts providerContactsForId(String id);
 
+  /** Return the parameters of the failed search. */
```

Comment:
Update this Javadoc, and any other documentation in the file that references mr. anderson or CDW resources.